pom.xml

```diff
@@ -85,11 +85,6 @@
             <artifactId>spring-boot-starter-validation</artifactId>
         </dependency>
 
-        <dependency>
```

Comment:
Why did you remove this starter here?

Reply:
The question is still without an answer...